sdk/storage/azblob/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.1 (Unreleased)
+## 1.2.0-beta.1 (Unreleased)
```

Comment:
I'll point out that the two bug fixes will be available in the beta only.  How important are they?  If you want to release them separately (recommended if customers are blocked on them) then this is what you should do.
- Create a branch `release/azblob-v1.1.1` from tag `sdk/storage/azcore/v1.1.0`
- Cherry-pick the commits for these bug fixes into the new branch
- Release `azblob@v1.1.1`
- Update the changelog in `main` to include the `v1.1.1` release notes
- Release `v1.2.0-beta.1`

This is what we've done for other packages (e.g. `azcore`) and it works well.  Just a bit of extra work to make sure the changelog in `main` is accurate.

Reply:
I don't think the bugs are critical, they can be released as a beta.  We will release these changes as a GA in about a month, following the workflow we use in the other languages.